js/taco.js

```diff
@@ -138,12 +138,13 @@ $(function() {
         update['/trucks/'+key] = {lat: lat, lng: lng};
         firebase.database().ref().update(update);
         location.hash = lat+'_'+lng;
-        $('#save').hide();
+        // $('#save').hide();
         // change icon
         activeMarker.setIcon(image);
         activeMarker.setAnimation(google.maps.Animation.BOUNCE);
         window.setTimeout(function() {
             activeMarker.setAnimation(null);
+            addMarker(new google.maps.LatLng(lat, lng), true);
```

Comment:
probably should put the new marker off to one side a little so it doesn't cover up the just-placed marker but we can fix that later